bin/windows/kafka-run-class.bat

```diff
@@ -110,13 +104,13 @@ IF ["%JMX_PORT%"] NEQ [""] (
 )
 
 rem Log directory to use
-IF ["%LOG_DIR%"] EQU [""] (
```

Comment:
Je me demandais si le "equals empty" aurait été changé récemment pour être plus générique que le "not defined". Ce qu'on veut vraiment est l'ajout des guillemets et ça donne l'impression qu'on a copié le fichier précédemment corrigé au lieu de réappliquer le même changement sur le nouveau fichier.

Reply:
Malgré l'impression que ça peut donner, c'est bel et bien un changement qu'on fait depuis la version 3.1 de Kafka. Le `IF ["%LOG_DIR%"] EQU [""]` est présent dans le code original de Kafka depuis de nombreuses années.